style/app.scss

```diff
@@ -755,6 +760,38 @@ div.sidebar.collapsed
   background-color : #cfccd4;
 }
 
+.label-hidden
+{
+  display: none;
+}
+
+.search-bar
+{
+  margin : 0 auto;
+}
+
+.search-dropdown
```

Comment:
Add a `list-style: none;` rule to this section, so that the "bullet" markers no longer appear